src/pages/DashboardPage/index.tsx

```diff
@@ -78,38 +104,44 @@ export default function DashboardPage() {
 
           <IoIosArrowForward className='text-3xl' />
         </div>
-        <div className=''>
+        <div className='text-center'>
           Go back to reading{' '}
           <span className='underline text-red-800 font-medium'>
             Vaping is Harmful
           </span>
         </div>
       </div>
-      <div className='mb-12 flex flex-col gap-2'>
-        <div className='text-2xl font-bold text-start'>Mindful Quest Games</div>
+      <div className='mb-12 flex flex-col gap-3'>
+        <div className='text-2xl font-bold text-start'>
+          Mindful Quest Games 🎮
+        </div>
         <div className='flex flex-row gap-4'>
           <GameThumbnail
             title='Be Drug ARware'
-            thumbnailImage={ARIcon}
-            imageClassName='h-[105%] w-[105%]'
+            thumbnailImage={arThumbnail}
+            imageClassName='h-full w-full'
           />
           <GameThumbnail
```

Comment:
I think would be better if this div has a cursor-pointer to it so that the user knows that it is clickable